examples/django/README.rst

```diff
@@ -34,3 +47,14 @@ Starting the worker
 .. code-block:: bash
 
     $ celery -A proj worker -l info
+
+Running a task
+===================
+
+.. code-block:: bash
+    
+    $ python ./manage.sh shell
```

Comment:
It should still be `manage.py` instead of `manage.sh` though :) And the leading `./` can be dropped if called by python directly.